bin/minifollowups/pycbc_page_snglinfo

```diff
@@ -72,20 +72,22 @@ elif args.n_loudest is not None:
                                       ranking_statistic=args.ranking_statistic)
     # Restrict to only the nth loudest, instead of all triggers up to nth
     # loudest
-    stat = sngl_file.stat[-1]
-    sngl_file.mask = sngl_file.mask[-1]
+    stat = sngl_file.stat
```

Comment:
Why make this change here? Aren't you just adding extra lines of code to do the same thing?

Reply:
I could merge line 75 and 76 if that's what you mean?? 

One difference here is that stat *cannot* be guaranteed to already be sorted, and in general it will not be. Part of the reason for this is constrained by what h5py can support as a directly slice operation. It can either take a fool boolean array or a list of values. In this case, it's a list. But for h5py to accept it, that list *must* be in ascending order of indices. No other order is allowed, so having it presorted by statistic value is not possible.